src/utils/utils.ts

```diff
@@ -59,3 +60,11 @@ export const regexMatch = (str: string, regexp: RegExp, index: number) => {
 }
 
 export const isDocument = (node: Node): node is Document => node.nodeType === Node.DOCUMENT_NODE
+
+export const batchReplace = (source: string, group: Array<[string | RegExp, string]>) => {
```

Comment:
Leave a comment for this function at twitter support stage 4.

Reply:
fixed 1456b506d53106f4fc51240ed5d2566ccfca465f